### PR TITLE
💄 [Dashboard/] Add total sales to dashboard

### PR DESCRIPTION
**Pull request by AuroraHuang22**
截圖 2023-03-03 上午10 08 15

截圖 2023-03-03 上午10 08 24

### CODE REVIEW

File: src/locales/zh-Hant.json [hunk 1]
```diff
@@ -333,6 +333,7 @@
   "nft_portfolio_page_state_collections": "藏品",
   "nft_portfolio_page_state_collectors": "收藏者",
   "nft_portfolio_page_state_creations": "作品",
+  "nft_portfolio_page_state_sales": "銷售額 (LIKE)",
```
Comment (williamchong) [inline src/locales/zh-Hant.json:336]:
```suggestion
  "nft_portfolio_page_state_sales": "總銷售額 (LIKE)",

```

File: src/locales/en.json [hunk 2]
```diff
@@ -346,6 +346,7 @@
   "nft_portfolio_page_state_collections": "Collections",
   "nft_portfolio_page_state_collectors": "Collectors",
   "nft_portfolio_page_state_creations": "Creations",
+  "nft_portfolio_page_state_sales": "Revenue",
```
Comment (williamchong) [inline src/locales/en.json:349]:
```suggestion
  "nft_portfolio_page_state_sales": "Total Revenue",

```

File: src/components/UserStats/MyDashboard.vue [hunk 3]
```diff
@@ -33,11 +33,9 @@
         </UserStatsItem>
         <UserStatsItem
           :is-loading="stats.isLoadingStats"
+          :stats-value="stats.collectedAmount"
           :label-text="$t('nft_portfolio_page_state_value')"
         >
-          <template #custom-value>
```
Comment (williamchong) [inline src/components/UserStats/MyDashboard.vue:38]: Why is this removed?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/components/UserStats/MyDashboard.vue (hunk 3 above)
Comment (AuroraHuang22) [reply src/components/UserStats/MyDashboard.vue:38]: I just formatted the `collectedAmount` in props ( also `createdTotalSales`), so there is no need to do the format here.

**Review by WeiJun0827 — APPROVED** [review]

File: src/locales/en.json [hunk 4]
```diff
@@ -346,6 +346,7 @@
   "nft_portfolio_page_state_collections": "Collections",
   "nft_portfolio_page_state_collectors": "Collectors",
   "nft_portfolio_page_state_creations": "Creations",
+  "nft_portfolio_page_state_sales": "Total Revenue",
```
Comment (williamchong) [inline src/locales/en.json:349]:
```suggestion
  "nft_portfolio_page_state_sales": "Total Revenue (LIKE)",

```
For consistency with zh version and asset value below

**Review by williamchong — CHANGES_REQUESTED** [review]

**Comment by williamchong** [comment]
i think we can target master instead of develop for this PR

**Review by williamchong — APPROVED** [review]

File: src/components/UserStats/Controller.vue [hunk 5]
```diff
@@ -35,14 +37,17 @@ export default {
       return this.userStats?.collectedClassCount || 0;
     },
     collectedAmount() {
-      return this.userStats?.collectedValue || 0;
+      return formatNumber(this.userStats?.collectedValue) || 0;
```
Comment (nwingt) [inline src/components/UserStats/Controller.vue:40]:
```suggestion
      return formatNumber(this.userStats?.collectedValue || 0);
```

File: src/components/UserStats/Controller.vue [hunk 6]
```diff
@@ -35,14 +37,17 @@ export default {
...
     },
     createdCount() {
       return this.userStats?.createdClassCount || 0;
     },
     createdCollectorCount() {
       return this.userStats?.createdCollectorCount || 0;
     },
+    createdTotalSales() {
+      return formatNumber(Math.floor(this.userStats?.createdTotalSales)) || 0;
```
Comment (nwingt) [inline src/components/UserStats/Controller.vue:49]:
```suggestion
      return formatNumber(Math.floor(this.userStats?.createdTotalSales || 0));
```

File: src/components/UserStats/MyDashboard.vue [hunk 7]
```diff
@@ -78,17 +80,21 @@
             <IconPersonMini />
           </template>
         </UserStatsItem>
+        <UserStatsItem
+          :is-loading="stats.isLoadingStats"
+          :stats-value="stats.createdTotalSales.toString()"
```
Comment (nwingt) [inline src/components/UserStats/MyDashboard.vue:85]: No need to cast string?

**Review by nwingt — APPROVED** [review]

**Review by williamchong — APPROVED** [review]
